### ISSUE TITLE
Basic access authentication possible

### ISSUE THREAD

**Issue by ghost** [dimkanovikov/WebLoader#11]
Is it possible to use Basic access authentication (https://en.wikipedia.org/wiki/Basic_access_authentication) with this library?

**Comment by dimkanovikov** [comment]
Yes with some improvements. You need to add a method in NetworkRequest which allow you to add user-defined header and then you can pass the Authorization header through this method.
